### PR TITLE
[5.0] [WebProfilerBundle] Add parameter type-hints where possible

### PR DESCRIPTION
**Pull request by Matts**
| Q             | A
| ------------- | ---
| Branch?       | 5.0
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no    
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | #32179
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Add type-hints for WebProfilerBundle in 5.0

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,725 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (stof) [inline .idea/workspace.xml]: this file must not be committed (look at https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore to see how to automatically ignore such files generated by your IDE)

**Review by stof — CHANGES_REQUESTED** [review]

File: .idea/workspace.xml (hunk 1 above)
Comment (Matts) [reply .idea/workspace.xml]: lol I usually remove this manually before committing. But thanks for the link for ignoring the files globally!

File: src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php [hunk 2]
```diff
@@ -156,6 +156,8 @@ public function toolbarAction(Request $request, $token)
     /**
      * Renders the profiler search bar.
      *
+     * @param Request $request
```
Comment (derrabus) [inline src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php]: This annotation is redundant and does not add information. Please remove it.

**Review by derrabus — CHANGES_REQUESTED** [review]

File: src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php (hunk 2 above)
Comment (Matts) [reply src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php]: Done, also did as you requested in https://github.com/symfony/symfony/pull/32218#discussion_r298189660

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @Matts.

**Pull request merged**